website/docs/docs/dbt-versions/core-upgrade/04-upgrading-to-v1.11.md

```diff
@@ -0,0 +1,97 @@
+
+We continue to recommend explicitly installing both `dbt-core` and `dbt-<youradapter>`. This may become required for a future version of dbt. For example:
+
+```sql
+python3 -m pip install dbt-core dbt-snowflake
+```
+
+## New and changed features and functionality
+
+New features and functionality available in <Constant name="core" /> v1.11
+
+### User-defined functions (UDFs) <Lifecycle status="beta" />
+
+dbt Core v1.11 introduces support for user-defined functions (UDFs), which enable you to define and register custom functions in your warehouse. Like macros, UDFs promote code reuse, but they run natively in the warehouse so you can reuse the same logic in tools outside dbt.
+
+Key features include:
+
+- **Define UDFs as first-class dbt resources**: Create UDF files in a `functions/` directory with corresponding YAML configuration.
+- **Native warehouse execution**: Create, update, and rename UDFs as part of DAG execution.
+- **DAG integration**: UDF files are built before models that reference them, ensuring proper dependency management.
+- **New `function()` macro**: Reference UDFs in your models using the `{{ function('function_name') }}` Jinja macro.
+- **Unit test support**: Validate models that use UDFs with unit tests. Make sure the functions exist by running `dbt build --select "+model_to_test" --empty` before running unit tests.
+- **List and select functions**: Manage UDFs by using `dbt list` and selection syntax like `dbt run --resource_type function` or `dbt run --select my_function`.
+
+Read more about UDFs, including prerequisites and how to define and use them in the UDF documentation.
+
+### Catalog improvements
+
+The catalog integration has been enhanced with new configuration options and validation:
+
+- `file_format` configuration is now supported in catalog integration.
+- Improved catalog and config validation for sources, models, and tests.
+- Better error handling and validation messages.
+
+### Enhanced `dbt ls` output
+
+The `dbt ls` command now supports nested key traversal. This makes it easier to query and filter complex metadata from your dbt project.
+
+### Manifest metadata enhancements
+
+Manifest metadata now includes `run_started_at`, providing better tracking of when dbt runs were initiated.
+
+### Improved config validation
+
+SQL-based configuration validation has been improved for models. This enables earlier detection of configuration errors during development.
+
+### Managing changes to legacy behaviors
+
+Starting with `v1.10`, you can [manage changes to legacy behaviors](/reference/global-configs/behavior-changes). You may opt into recently introduced changes (disabled by default), or opt out of mature changes (enabled by default), by setting `True` / `False` values, respectively, for `flags` in `dbt_project.yml`.
+
+### Deprecation warnings
```

Comment:
this was for 1.10 - i don't think we need to call it out for 1.11

Reply:
unless we've switched the default? and now turned out deprecation warnings from json schema validation for 1.11? if so, that should be the explicit callout

Reply:
@graciegoheen  I wasn't sure if we'd want to continue to help folks through deprecations. 

@QMalcolm what is the state of deprecation warnings in the code? Happy to capture whatever users will need help with here.

Reply:
I will remove now and add in our [1.11 GA jira](https://dbtlabs.atlassian.net/browse/PRODDOCS-244)